src/pages/progress_reports/SubmitProgressReport.tsx

```diff
@@ -5,35 +5,128 @@ import {Notifications} from 'src/libs/utils';
 import {ConsentError} from 'src/types/responseTypes';
 import {validationFailed} from "src/utils/darFormUtils";
 import {FormValidationState} from 'src/pages/dar_application/FormValidationState';
+import {DMI_INCIDENT_KEYS, FormState} from "src/pages/progress_reports/ProgressReportFormState";
+import {PublicationOrPresentation} from "src/components/publications_list/PublicationOrPresentation";
+import {
+  DataAccessRequest,
+  DataManagementIncident,
+  Presentation,
+  Publication
+} from "src/types/model";
 
 
 interface SubmitProgressReportProps {
-  readonly progressReport: object;
+  readonly formState: FormState;
   readonly parentReferenceId: string;
   readonly onSuccess: (result: unknown) => void;
-  readonly onCancel: (result: unknown) => void;
+  readonly onCancel: () => void;
   readonly validateForm: () => FormValidationState;
 }
 
 export default function SubmitProgressReport(props: SubmitProgressReportProps) {
-  const {progressReport, parentReferenceId, onSuccess, onCancel, validateForm} = props;
+  const { formState, parentReferenceId, onSuccess, onCancel, validateForm} = props;
+
+  const getPublicationList = (formState: FormState): Publication[] => {
+    const publications: PublicationOrPresentation[] = formState.publications ?? [];
+    return publications.map((pub: PublicationOrPresentation) => {
+      const expectedPublication: Publication = {} as Publication;
+      expectedPublication.title = pub.title;
+      expectedPublication.pubmedId = pub.pubmed_id;
+      expectedPublication.date = pub.date;
+      expectedPublication.authors = pub.authors;
+      expectedPublication.bibliographicCitation = pub.bibliographic_citation
+      expectedPublication.datasetCitation = pub.dataset_citation;
+      expectedPublication.citation = pub.did_cite;
+      return expectedPublication;
```

Comment:
Another solution here might be to refactor these values so they match what is expected, that way no transformation is required. (I can do this if you'd like)

Reply:
That would definitely be better, and I think breaking it out into a follow-on ticket/PR would be good.